web/src/pages/CustomerSatisfactionPage/CustomerSatisfactionPage.tsx

```diff
@@ -0,0 +1,56 @@
+import { Container, Stack, Text } from '@chakra-ui/react'
+import { useState } from 'react'
+
+import CsatCell from 'src/components/CsatCell/CsatCell'
+import NpsCell from 'src/components/NpsCell/NpsCell'
+import UeqCell from 'src/components/UeqCell/UeqCell'
+import DemographicCell from 'src/components/DemographicCell/DemographicCell'
+import { navigate, routes } from '@redwoodjs/router'
+import { useAuth } from 'src/auth'
+
+
+
+const CustomerSatisfactionPage = () => {
+  const { currentUser } = useAuth()
+  const [step, setStep] = useState(1)
+  const handleNextStep = () => {
+    setStep((s) => s + 1)
+    // navigate(routes.home(), {replace: true})
+  }
+  return (
+    <Container maxW="6xl">
+    <Stack direction="column" gap={8} alignItems="center">
+      <Text data-testid="instruction">
+        You are shown a picture and please answer some questions
+      </Text>
```

Comment:
Remove ?

Reply:
We still need context and instruction on each page, but I need to update this